catalog.spec

```diff
@@ -284,5 +284,40 @@ module Catalog {
     funcdef approve_developer(string username) returns () authentication required;
     funcdef revoke_developer(string username) returns () authentication required;
 
+    /*
+        user_id - GlobusOnline login of invoker,
+        method_name - fully qualified method name (including module name and slash),
+        creation_time, exec_start_time and finish_time - defined in milliseconds.
+    */
+    typedef structure {
+        string user_id;
+        string method_name;
+        int creation_time;
+        int exec_start_time;
+        int finish_time;
+        boolean is_failure;
+    } LogExecutionStatsParams;
```

Comment:
Should we have a method commit hash for SDK methods?  To start with we will probably aggregate over all versions, but we might want to eventually split things up based on versions, especially if/when we allow running old versions.